Source/Controllers/ESTabBarController.m

```diff
@@ -233,12 +253,17 @@ - (void)setupInterface {
 - (void)setupButtons {
     if (self.buttons == nil) {
         self.buttons = [NSMutableArray arrayWithCapacity:self.tabIcons.count];
+        self.badgeLabels = [NSMutableArray arrayWithCapacity:self.tabIcons.count];
         
         for (NSInteger i = 0; i < self.tabIcons.count; i++) {
             UIButton *button = [self createButtonForIndex:i];
             
             [self.buttonsContainer addSubview:button];
             self.buttons[i] = button;
+            
+            UILabel *badgeLabel = [self createBadgeLabel];
```

Comment:
Would be nice if we can take the badges setup to another method, say `- (void)setupBadeLabels`.